tests/integration/test_data/cached_dependencies.yaml

```diff
@@ -100,19 +100,100 @@ cached_deps:
+    - "pkg:generic/appr?download_url=https%3A%2F%2Fgithub.com%2Fquay%2Fappr%2Farchive%2F37ff9a487a54ad41b59855ecd76ee092fe206a84.zip%23egg%3Dappr%26cachito_hash%3Dsha256%3Aee6a0a38bed8cff46a562ed3620bc453141a02262ab0c8dd055824af2829ee5c&checksum=sha256:ee6a0a38bed8cff46a562ed3620bc453141a02262ab0c8dd055824af2829ee5c"
+    - "pkg:github/cachito-testing/cachito-pip-without-deps@SECOND_DEP_COMMIT"
+    - "pkg:github/quay/appr@58c88e4952e95935c0dd72d4a24b0c44f2249f5b"
+    - "pkg:pypi/aiowsgi@0.7"
+# Test gomod with cached dependencies
+gomod_cached_deps:
+  # Use local version of repos.
+  # <use_local: True> is not supported in local environment and will be skipped.
+  use_local: True
+  # test repo user
+  git_user: "Arthur Dent"
+  # test repo user email
+  git_email: "dent42@cachito.rocks"
+  # HTTPS and SSH links to main and dependency repos respectively
+  https_main_repo: https://github.com/cachito-testing/cachito-gomod-with-deps.git
+  https_dep_repo: https://github.com/cachito-testing/cachito-gomod-without-deps.git
+  ssh_main_repo: "git@github.com:cachito-testing/cachito-gomod-with-deps.git"
+  ssh_dep_repo: "git@github.com:cachito-testing/cachito-gomod-without-deps.git"
+  pkg_managers: [ "gomod" ]
+  # Parts of the Cachito response to check
+  response_expectations:
+    packages:
+    - dependencies:
+      - name: "github.com/cachito-testing/cachito-gomod-without-deps"
+        replaces: null
+        type: "gomod"
+        version: "DEP_VERSION"
+      name: "github.com/cachito-testing/cachito-gomod-with-deps"
+      type: "gomod"
+      version: "MAIN_VERSION"
+    - dependencies:
+      - name: "rsc.io/quote"
+        replaces: null
+        type: "go-package"
+        version: "v1.5.2"
+      - name: "rsc.io/sampler"
+        replaces: null
+        type: "go-package"
+        version: "v1.3.0"
+      - name: "golang.org/x/text/language"
+        replaces: null
+        type: "go-package"
+        version: "v0.0.0-20170915032832-14c0d48ead0c"
+      - name: "golang.org/x/text/internal/tag"
+        replaces: null
+        type: "go-package"
+        version: "v0.0.0-20170915032832-14c0d48ead0c"
+      name: "github.com/cachito-testing/cachito-gomod-with-deps"
+      type: "go-package"
+      version: "MAIN_VERSION"
+    dependencies:
+    - name: "golang.org/x/text/internal/tag"
+      replaces: null
+      type: "go-package"
+      version: "v0.0.0-20170915032832-14c0d48ead0c"
+    - name: "golang.org/x/text/language"
+      replaces: null
+      type: "go-package"
+      version: "v0.0.0-20170915032832-14c0d48ead0c"
+    - name: "rsc.io/sampler"
+      replaces: null
+      type: "go-package"
+      version: "v1.3.0"
+    - name: "rsc.io/quote"
+      replaces: null
+      type: "go-package"
+      version: "v1.5.2"
+    - name: "github.com/cachito-testing/cachito-gomod-without-deps"
+      replaces: null
+      type: "gomod"
+      version: "DEP_VERSION"
+  # Expected files
+  expected_files: []
```

Comment:
I'm not sure I understand why the expected files cannot be checked, can you explain more specifically?

Reply:
@chmeliik Source files and dependencies have pseudo-version in files and file names. We can check files, but I need to change dependency filenames and source file content.

Reply:
Ohh, makes sense, I thought it was only the filenames. I didn't realize the pseudo-versions were also included in the file content.